src/components/Shuffle/index.js

```diff
@@ -0,0 +1,6 @@
+const Shuffle = (data) => {
+  const shuffled = data.sort(() => 0.5 - Math.random())
+  let shuffledData = shuffled.slice(0, 12)
```

Comment:
We should add a parameter here

```suggestion
  let shuffledData = shuffled.slice(0, maxItems)
```

Reply:
the parameter has been added

Reply:
> I added some styling for the dark theme - at this stage we should not become crazy for these stuff
> 
> <img alt="image" width="439" src="https://user-images.githubusercontent.com/11404065/212746514-b59d9b2f-6df9-4828-9e45-e7652bdb6242.png">
> 
> I will add your suggestion to the next project: an org should be able to add a logo for the dark theme.
> 
> Since I think it is valuable to ship the home page header, I would proceed in this way:
> 
>     * clean the code and remove all sections that are not completed (tracked here [#38 (comment)](https://github.com/fastify/website-next/pull/38#issuecomment-1376332179) )
> 
>     * last review and merge it
> 
>     * create a PR for each section
> 
> 
> This would ease a lot the review for both. Do you agree?

thank you for the styling. okay please, i agree let me do a clean up. would creating a pr for each section mean i delete all other sections in the first place? since some of these having shared components and styles wouldn't it be better to clean up and no broken links etc for everything?

Reply:
> delete all other sections in the first place?

Yes, what I would do:

1. create a new branch from the actual status: `git checkout -b temp`
2. then switch back to this branch: `git checkout index-page`
3. then delete the things from the "core features" section and all the unused files (like the code examples)
4. commit and push the cleaned version

Then we can merge it and we can add section by section by picking the code from the `temp` branch - in this way we don't lose the code

Reply:
alright, i have done so. will be opening a pr for every section progressively until we achieve the entire home page